src/main/java/callofcactus/map/DefaultMap.java

```diff
@@ -27,11 +30,10 @@ public Map(Game currentGame, int currentWidth, int currentHeight) {
 		this.currentHeight = currentHeight;
 		widthRatio = (double) currentWidth / width;
 		heightRatio = (double) currentHeight / height;
-
-		initAllWalls();
 	}
 
-	private void initAllWalls() {
+	@Override
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method init() has an NPath complexity of 256](https://www.codacy.com/app/guushamm/Call-of-Cactus/file/1385244491/issues/source?bid=2979575&fileBranchId=2980149#l36)